model/trace/instrumentation/gcp-client.yml

```
@@ -0,0 +1,12 @@
+groups:
+- id: gcp.client.scope
+  type: scope
```

please don't use `scope` type, it's going to be removed. Also, many languages don't support scope attributes and the only way to report it would be through regular span/metric/event attribute.

```suggestion
  type: attribute_group
```

---

Thanks! Good to know that this is being removed. Updated to use `attribute_group` instead of `scope`. Also changed the `id` to not reference `scope` given this change in type.